src/EnergyPlus/DataGlobalConstants.hh

```diff
@@ -242,6 +242,11 @@ namespace DataGlobalConstants {
     extern int const iEvapCoolerInDirectRDDSpecial;
     extern int const iEvapCoolerDirectResearchSpecial;
 
+    extern std::string FuelTypeInput;
+    extern std::string FuelType;
+    extern int FuelTypeNum;
+    extern bool FuelTypeErrorsFound;
```

Comment:
I do not understand the need for these global variables.  That new function accepts a `FuelTypeInput` argument.  What is this global variable `FuelTypeInput` for?  I think you need to add two arguments to the function to return back the FuelTypeNum and FuelType values.  No need to add global variables.